### PR TITLE
Refactor MapRenderer.

### PR DESCRIPTION
**Pull request by siarheishut**
Change `svg` library tag to `v2.0` in order to use svg::Rectangle and svg::Section. 
Change method name from GetMap to RenderMap, because Getter is meant to have O(1) complexity and very simple logic, while this method no longer meets these requirments. 
Change field `map_` type from svg::Document to svg::Section in order not to render it on every Map call. 
Remove extra parameter in render functions and add SectionBuilder parameter to them.

### CODE REVIEW

File: src/map_renderer.h [hunk 1]
```diff
@@ -22,35 +22,38 @@ class MapRenderer {
...
               const RenderingSettings &settings);
 
   void AddBusLinesLayout(
+      svg::SectionBuilder &builder,
       const renderer_utils::Buses &buses,
-      const renderer_utils::Stops &stops,
       const rm::RenderingSettings &settings,
       const renderer_utils::StopCoords &coords);
 
   void AddBusLabelsLayout(
```
Comment (foxfrikses) [inline src/map_renderer.h:38]: Shouldn't these functions then become const, as they don't modify and class data?
Do they even use and class data?

Comment (foxfrikses) [reply src/map_renderer.h:38]: If no, shouldn't they then be static methods or even separate functions?

File: src/map_renderer.cpp [hunk 2]
```diff
@@ -203,56 +208,60 @@ void MapRenderer::AddBusLabelsLayout(
...
+                              .SetOffset(settings.stop_label_offset)
+                              .SetFontSize(settings.stop_label_font_size)
+                              .SetFontFamily("Verdana")
+                              .SetData(std::string(stop))
+                              .SetFillColor("black")));
   }
 }
 
-std::string MapRenderer::GetMap() const {
+std::string MapRenderer::Map() const {
```
Comment (foxfrikses) [inline src/map_renderer.cpp:260]: Renaming from 'GetMap' to 'Map' lefts it a getter anyway. If you want to specifically underline that it does job to prepare the data, you may name it 'RenderMap' or anything idk

Comment (siarheishut) [reply src/map_renderer.cpp:260]: Changed to `RenderMap`.

File: src/map_renderer.h (hunk 1 above)
Comment (siarheishut) [reply src/map_renderer.h:38]: Made them static.

File: src/map_renderer.h [hunk 3]
```diff
@@ -22,35 +22,38 @@ class MapRenderer {
...
       const renderer_utils::StopCoords &coords);
 
   void AddStopLabelsLayout(
+      svg::SectionBuilder &builder,
       const renderer_utils::Stops &stops,
       const rm::RenderingSettings &settings,
       const renderer_utils::StopCoords &coords);
 
-  svg::Document map_;
+  svg::Section map_ = svg::SectionBuilder{}.Build();
```
[the remark this answers is not in the document]
Comment (siarheishut) [reply src/map_renderer.h:56]: Added the initializer list instead.

**Pull request merged**